pkgs/misc/scrcpy/default.nix

```diff
@@ -22,8 +22,14 @@ stdenv.mkDerivation rec {
     owner = "Genymobile";
     repo = "scrcpy";
     rev = "v${version}";
-    sha256 = "02szi8w3w0lacyz42hlayxififi863qpm63yg9qir3jcl2vs7vdk";
+    sha256 = "015j1h161c8s2vyh5rbpj0q473zbp5sygw5y7cyv59237d703kcb";
   };
+  postPatch = ''
+    substituteInPlace app/src/screen.c \
+      --replace "SDL_RENDERER_ACCELERATED" "SDL_RENDERER_ACCELERATED || SDL_RENDERER_SOFTWARE"
```

Comment:
What is the effect of this patch?

Reply:
When run without a hardware accelerator, this allows the command to continue working rather than failing unexpectedly. This can happen when running on non-NixOS because then scrcpy seems to have a hard time using the host OpenGL-supporting hardware. It would be better to fix the OpenGL problem, but that seems much more intrusive.